main/settings.py

```diff
@@ -27,9 +28,10 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = os.environ.get("SECRET_KEY", "development_secret_key")
 
-DEBUG = True
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = os.environ.get("DJANGO_DEBUG") != "False"
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["anitab-forms-backend.herokuapp.com"]
```

Comment:
@Rahulm2310 I think this is deployed in your Heroku account. So adding it here wouldn't be a nice logic.